geomesa-spark/geomesa-spark-sql/src/test/scala/org/locationtech/geomesa/spark/SparkSQLGeometryProcessingFunctionsTest.scala

```diff
@@ -43,31 +43,45 @@ class SparkSQLGeometryProcessingFunctionsTest extends Specification with LazyLog
+          """.stripMargin)
+        buf.
```

Comment:
Looks like IntelliJ made some weird decisions when it reformatted the code! Fixed.